presto-main/src/main/java/io/prestosql/sql/planner/iterative/rule/PushPredicateIntoTableScan.java

```diff
@@ -90,20 +90,14 @@ public PickTableLayout(Metadata metadata, SqlParser parser)
     {
         return ImmutableSet.of(
                 checkRulesAreFiredBeforeAddExchangesRule(),
```

Comment:
is this still required? I would replace with comment in `PlanOptimizers`. There is no reason to prevent this class to be invoked later.

Reply:
Probably not. I'll remove it.